core/tauri-utils/src/lib.rs

```diff
@@ -294,3 +297,12 @@ macro_rules! debug_eprintln {
     $crate::consume_unused_variable!($($arg)*);
   };
 }
+
+/// Reconstructs a path from its components using the platform separator then converts it to String
+pub fn display_path<P: AsRef<Path>>(p: P) -> String {
+  p.as_ref()
+    .components()
+    .collect::<PathBuf>()
+    .to_string_lossy()
```

Comment:
shouldn't we return a .display() here instead?

Reply:
right, pushed it